README.md

```diff
@@ -586,13 +586,16 @@ The IP configuration supports the following options:
 
   Static route configuration can be specified via a list of routes given in the
```

Comment:
regarding the commit message, seems the title can be simplified as:

"feat: Support blackhole, prohibit and unreachable route types"

I am pretty sure the "types" needs to be plural.

Regarding the details in the commit message, it is missing that this is only for the nm provider using network_connections. Is it already supported via network_state, then it could mention this as well. The detailed explanations for the different types are hard to understand. I am not convinced that it needs to be in that detail as well. If you want that detail, please make it easier to understand. Otherwise the key message is that the types are useful in BGP setups which could be enough.

If we have a Jira ticket for this, maybe it is also good practice to mention it as resolves.